modules/cli/pages/custom-certs.adoc

```diff
@@ -0,0 +1,143 @@
+----
+
+.. Copy the generated `tls.crt` from the `ca` directory to the `certificate` directory`:
++
+----
+$ cp ../ca/tls.crt ./ca.crt
+----
+
+
+You should now have a root certificate in the `ca` directory and another certificate in the `certificate` directory that you can use with a site.
+
+
+--
+
+. Create secrets for the site
++
+--
+.. Change to the parent directory of the `certificate` directory:
++
+----
+$ cd ..
+----
+
+.. Populate the `ca` related secrets using the certificate from the `ca` directory:
++
+----
+$ kubectl create secret tls skupper-site-ca --cert=ca/tls.crt --key=ca/tls.key
+
+$ kubectl create secret tls skupper-service-ca --cert=ca/tls.crt --key=ca/tls.key
+
+$ kubectl create secret tls skupper-local-ca --cert=ca/tls.crt --key=ca/tls.key
+
+----
+
+.. Populate the other secrets and modify them into the format required by `skupper`:
++
+----
+$ kubectl create secret tls skupper-claims-server --cert=certificate/tls.crt --key=certificate/tls.key 
+
+$ kubectl patch secret skupper-claims-server  -p="{\"data\":{\"ca.crt\": \"$($ kubectl get secret skupper-site-ca -o json -o=jsonpath="{.data.tls\.crt}")\"}}"
+
+
+$ kubectl create secret tls skupper-console-certs --cert=certificate/tls.crt --key=certificate/tls.key 
+
+$ kubectl patch secret skupper-console-certs  -p="{\"data\":{\"ca.crt\": \"$($ kubectl get secret skupper-local-ca -o json -o=jsonpath="{.data.tls\.crt}")\"}}"
+
+
+$ kubectl create secret tls skupper-local-client --cert=certificate/tls.crt --key=certificate/tls.key 
+
+$ kubectl patch secret skupper-local-client  -p="{\"data\":{\"ca.crt\": \"$($ kubectl get secret skupper-local-ca -o json -o=jsonpath="{.data.tls\.crt}")\"}}"
+
+
+$ kubectl create secret tls skupper-local-server --cert=certificate/tls.crt --key=certificate/tls.key 
+
+$ kubectl patch secret skupper-local-server  -p="{\"data\":{\"ca.crt\": \"$($ kubectl get secret skupper-local-ca -o json -o=jsonpath="{.data.tls\.crt}")\"}}"
+
+
+$ kubectl create secret tls skupper-site-server --cert=certificate/tls.crt --key=certificate/tls.key 
+
+$ kubectl patch secret skupper-site-server  -p="{\"data\":{\"ca.crt\": \"$($ kubectl get secret skupper-site-ca -o json -o=jsonpath="{.data.tls\.crt}")\"}}"
+
+
+$ kubectl create secret tls skupper-service-client --cert=certificate/tls.crt --key=certificate/tls.key 
+
+$ kubectl patch secret skupper-service-client  -p="{\"data\":{\"ca.crt\": \"$($ kubectl get secret skupper-service-ca -o json -o=jsonpath="{.data.tls\.crt}")\"}}"
+----
+
+--
+
+. Create the site using the following command:
++
+--
+----
+$ skupper init
+----
+On OpenShift, `skupper` defaults to use the `route` ingress, which is the equivalent of `skupper init --ingress route`.
+
+To verify your site, check the status:
+----
+$ skupper status
+----
+
+--
```

Comment:
This is great!
Now I'm thinking that maybe would be helpful to add some command for troubleshooting, like:
`skupper debug events` --> if some cert is not working a message is going to appear here
`oc get routes` --> to check if the routes included in as AltNames match the ones created by Skupper

What do you think?

Reply:
yes, i put those suggestions in